src/obr/signac_wrapper/operations.py

```diff
@@ -231,7 +242,11 @@ def initialize_if_required(job: Job) -> bool:
         copy_instead_link = job.sp().get("operation") == "shell"
         _link_path(base_path, dst_path, parent_id, copy_instead_link)
         job.doc["state"]["is_initialized"] = True
-        logging.info(f"Done initialization of case {job.id}")
+        if GLOBAL_UNINIT_COUNT:
+            logging.info(
+                "Done initialization of case"
+                f" {job.id} [{GLOBAL_INIT_COUNT}/{int(GLOBAL_UNINIT_COUNT)}]\r"
```

Comment:
Consider adding more information so that a user understands what that fraction means